app/views/layouts/dashboard/_sidebar.html.erb

```diff
@@ -3,8 +3,8 @@
     <!-- Brand logo -->
     <p class="navbar-brand flex-column">
       <%= link_to home_index_path, class: "nav-link" do %>
-        <% if Current.organization.avatar.attached? %>
-          <%= image_tag Current.organization.avatar, alt: current_organization_name, title: current_organization_name, class: 'rounded-1' %>
+        <% if Current.organization.avatar.attached? && Current.organization.avatar.persisted? %>
```

Comment:
without this if you upload a logo on the org and it fails validation we get an error